device/core/src/module_client.ts

```diff
@@ -222,4 +222,25 @@ export class ModuleClient extends EventEmitter {
   static fromAuthenticationProvider(authenticationProvider: AuthenticationProvider, transportCtor: any): ModuleClient {
     return InternalClient.fromAuthenticationProvider(authenticationProvider, transportCtor, ModuleClient) as ModuleClient;
   }
+
+  /**
+   * @description         Creates an IoT Hub device client by using configuration
```

Comment:
> IoT Hub device [](start = 36, length = 15)

Edge module client?

Reply:
or just module client?

---
In reply to: [194502582](https://github.com/Azure/azure-iot-sdk-node/pull/310#discussion_r194502582) [](ancestors = 194502582)

Reply:
Fixed.